src/main/fc/fc_init.c

```diff
@@ -478,8 +478,9 @@ void init(void)
     initBoardAlignment(boardAlignment());
 
     if (!sensorsAutodetect()) {
-        // if gyro was not detected due to whatever reason, we give up now.
-        failureMode(FAILURE_MISSING_ACC);
+        // if gyro was not detected due to whatever reason, notify and don't arm.
+        failureLedCode(FAILURE_MISSING_ACC, 2);
```

Comment:
Hmm... I'm with you. Let me see if we can do something.

What's bugging me most is that it will cause the initial flashing to take little less than a minute to complete waiting for the led sequence to finish.

- Simply get along with it. Configurator and the user can wait.

- If we can create a variable that is set when hardware is fully configured, we can distinguish the error case from un-configured case.